README.md

```diff
@@ -58,14 +58,6 @@ Run the project locally:
 bun run dev
 ```
 
-## Local Preview 👀
-
-To locally preview the production build, first run `bun run build` to build the project, then run the following:
-
-```shell
-bun run preview
-```
-
```

Comment:
Developers who do not have the NUXT_UI_PRO_LICENSE will not be able to successfully build and preview the project locally. To prevent any confusion, I am removing this.